### ISSUE TITLE
Use valid XHTML syntax

### ISSUE THREAD

**Issue by pmoura** [readthedocs/sphinx_rtd_theme#985]
The following lines:

https://github.com/readthedocs/sphinx_rtd_theme/blob/1d856256676d2b801a12c30e4a849c5abc466da9/sphinx_rtd_theme/layout.html#L15-L17

should be:

```html
<meta charset="utf-8" /> 
{{ metatags }} 
<meta name="viewport" content="width=device-width, initial-scale=1.0" /> 
```

**Comment by utzig** [comment]
Not really, `<meta>` tags are empty elements: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta

**Comment by pmoura** [comment]
Without these fixes we get invalid EPUB files due to invalid **XHTML** (not HTML!) files.

**Comment by pmoura** [comment]
P.S. Same fixes have already been applied to the themes bundled with Sphinx (in July 17).

**Comment by pmoura** [comment]
In the just released Sphinx Release 3.2.0, the bug fix is listed as:

#7846: html theme: XML-invalid files were generated
